modules/sdk-coin-xrp/src/xrp.ts

```diff
@@ -318,7 +332,7 @@ export class Xrp extends BaseCoin {
         fee: {
           fee: transaction.Fee,
           feeRate: null,
-          size: txHex.length / 2,
+          size: txHex!.length / 2,
```

Comment:
Is there an issues with floats or integer division here?

```suggestion
size: txHex.length / 2,
```